tests/tools/onert_train/src/nnfw_util.h

```diff
@@ -32,6 +33,7 @@ namespace onert_train
 {
 uint64_t num_elems(const nnfw_tensorinfo *ti);
 uint64_t bufsize_for(const nnfw_tensorinfo *ti);
-} // end of namespace onert_train
 
+std::ostream &operator<<(std::ostream &os, const nnfw_train_info &info);
```

Comment:
### Background 

`nnfw_train_info` is a structure that appears to API-user, to set/get the training parameters.

After importer implemented, There are many ways to set training parameters : 
   * using cmd line input
   * using model file (metadata include traininginfo)
   * if nothing given, default parameter will be used

So, I'd like to
   * print training parameter in onert_train to show which one is selected
   * hide detailed print-logic to keep main function simple -> so, hide detailed things in nnfw_util.c

Reply:
@jyoungyun Could you share your opinion about this PR?